src/reactviews/pages/QueryResult/resultGrid.tsx

```diff
@@ -132,7 +140,7 @@ const ResultGrid = forwardRef<ResultGridHandle, ResultGridProps>(
             }
         };
 
-        useEffect(() => {
+        const createTable = () => {
```

Comment:
What's this change about? Any difference from the original version?

Reply:
Just putting this into a function to make the code more readable.